cli/src/bin/phylum/commands/auth.rs

```diff
@@ -36,6 +36,18 @@ pub fn handle_auth_status(config: &Config) {
     }
 }
 
+/// Display the current authentication token to the user, if one exists.
+pub fn handle_auth_token(config: &Config) {
+    match config.auth_info.offline_access.clone() {
+        Some(token) => {
+            print_user_success!("{}", token);
+        },
+        None => {
+            print_user_warning!("User is not currently authenticated, you can login with phylum auth login");
```

Comment:
am just wondering if we could phrase this warning a bit better - like `Missing authentication, please use  phylum auth login` or some such

Reply:
I don't have strong opinions on the warning, both options seem clear enough to me. I think we might want to delineate where the command starts though. For example: 

```
...you can login with `phylum auth login`
```

Reply:
yeah, it's totally not important - just the mixture of third person, and second person feels a bit off - since the warning is for the `user`